TheMovieDb/Network/MovieCombineClient.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  MovieCombineClient.swift
+//  TheMovieDb
+//
+//  Created by Alex on 30/11/21.
+//
+
+import Foundation
+import os.log
```

Comment:
Same as above, I don't see where are we using the logger in this file, maybe we can omit this import.

Reply:
Got it.